src/layout/IHyprLayout.cpp

```diff
@@ -443,6 +451,67 @@ void IHyprLayout::onMouseMove(const Vector2D& mousePos) {
     if (g_pInputManager->dragMode == MBIND_MOVE) {
 
         CBox wb = {m_vBeginDragPositionXY + DELTA, DRAGGINGWINDOW->m_vRealSize.goal()};
+        if (*SNAPENABLED && !DRAGGINGWINDOW->m_bDraggingTiled) {
```

Comment:
arent these two if blocks the same? can't we combine them?

(this and the one further down)

Reply:
If `SNAPWINDOWGAP` is zero, there's still `SNAPMONITORGAP` a bit further down.

My thinking was that `enabled` would act as the master switch, and then different snapping methods could be turned on or off with zero/non-zero values.

Reply:
no I mean this entire block and the one at line 577

Reply:
This one is for moving and the one at line 577 is for resizing. They're similar, admittedly. I'm currently working on some refactoring to make both sections more brief.